pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StreamAvroIntoKafkaCommand.java

```diff
@@ -104,8 +102,12 @@ public boolean execute()
     properties.put("serializer.class", "kafka.serializer.DefaultEncoder");
     properties.put("request.required.acks", "1");
 
-    ProducerConfig producerConfig = new ProducerConfig(properties);
-    Producer<byte[], byte[]> producer = new Producer<byte[], byte[]>(producerConfig);
+    StreamDataProducer streamDataProducer;
+    try {
+      streamDataProducer = StreamDataProvider.getStreamDataProducer(KafkaStarterUtils.KAFKA_09_PRODUCER_CLASS_NAME, properties);
```

Comment:
why keep `09` in the name? I thought these same files are going to be able to handle other versions as well?